src/FSharp.Data.GraphQL.Client/GraphQLClient.fs

```diff
@@ -72,7 +72,7 @@ module GraphQLClient =
     }
 
     /// Sends a request to a GraphQL server.
-    let sendRequest client request = (sendRequestAsync CancellationToken.None client request).Result
+    let sendRequest client request = (sendRequestAsync CancellationToken.None client request).GetAwaiter().GetResult()
```

Comment:
I think it does the same under the covers.

Reply:
`GetAwaiter().GetResult()` unwraps exception from `AggregateException`